runtime/kusama/src/lib.rs

```diff
@@ -1019,7 +1235,8 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllModules,
-	CustomOnRuntimeUpgrade
+	// TODO: CustomOnRuntimeUpgrade is done, lets just remove it here?
```

Comment:
I think we leave it there just to make it easier to hack such logic in the future.